src/applications/common/schemaform/save-in-progress/actions.js

```diff
@@ -196,23 +204,32 @@ export function saveAndRedirectToReturnUrl(...args) {
  *                                version of the form the data was saved with
  *                                is different from the current version.
  */
-export function fetchInProgressForm(formId, migrations, prefill = false, prefillTransformer = null) {
+export function fetchInProgressForm(formId, migrations, prefill = false, prefillTransformer = null, prestartForm = null) {
+
   // TODO: Migrations currently aren’t sent; they’re taken from `form` in the
   //  redux store, but form.migrations doesn’t exist (nor should it, really)
-  return (dispatch, getState) => {
+  return async (dispatch, getState) => {
+    if (prestartForm) {
```

Comment:
Same here. Seems like we should `await prestartForm` before calling `fetchInProgressForm`.

Reply:
@jbalboni recommended handling the prestart gating within `fetchInProgressForm`.